### PR TITLE
Bumped to version 2.0.0

### PR DESCRIPTION
**Pull request by metalshark**
Removed dependencies on API Manager and PacketHandler, moving it on to ProtocolLib instead.

### CODE REVIEW

File: .github\FUNDING.yml [hunk 1]
```diff
@@ -1,3 +0,0 @@
-github: InventivetalentDev
```
Comment (metalshark) [inline .github\FUNDING.yml]: The filename has a backwards slash in it causing issues. There's already a FUNDING.yml in .github.

Comment (metalshark) [reply .github\FUNDING.yml]: Same with stale.yml.

**Comment by InventivetalentDev** [comment]
While I appreciate the effort that went into these changes, I'm not sure if I'm comfortable merging them into master.  
I'll keep this here though and consider creating a separate version for users preferring ProtocolLib .

**Comment by metalshark** [comment]
> While I appreciate the effort that went into these changes, I'm not sure if I'm comfortable merging them into master.
> I'll keep this here though and consider creating a separate version for users preferring ProtocolLib .

No worries. Don't mind maintaining it as a fork/alternative version. If I tidy up the documentation, etc feel free to pull those in.

**Pull request closed without merging**